Makefile

```diff
@@ -0,0 +1,6 @@
+install:
+	python onelinerizer/__main__.py .setup.py setup.py
```

Comment:
`python -m onelinerizer .setup.py setup.py` is a bit more correct (would be required if we used relative imports).